etddf_minau/AssetEstimate[] assets